cpp/__init__.py

```diff
@@ -34,15 +34,15 @@ def cpp_types_decode(key):
     except KeyError:
         cpp_type = _cpp_types_common[key]
     if cpp_type == "NA":
-        raise NotImplementedError("Missing type Mapping")
+        raise NotImplementedError("Missing type Mapping for " + key)
     return cpp_type
 
 
 def get_size(type):
     try:
         size = _type_size[type]
     except KeyError:
-        raise NotImplementedError("Missing type size mapping")
+        raise NotImplementedError("Missing type size mapping " + type)
```

Comment:
```suggestion
        raise NotImplementedError("Missing type size Mapping " + type)
```

turbo nit - consistency with the other methods' exceptions.